java11/src/main/java/feign/http2client/Http2Client.java

```diff
@@ -44,7 +44,7 @@ public Http2Client() {
   }
 
   public Http2Client(HttpClient client) {
-    this.client = Util.checkNotNull(client, "http cliet must be not unll");
+    this.client = Util.checkNotNull(client, "HttpClient must be not null");
```

Comment:
FYI, proper English grammar is "must not be null" or "must be non null" (I'm not sure if grammar requires a hyphen in "non-null").

Reply:
Rephrased 50d209e